### PR TITLE
refactor: Migrate CopyToClipboard to TypeScript

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: src/script/components/copyToClipboard.ts [hunk 1]
```diff
@@ -18,14 +18,17 @@
  */
 
 class CopyToClipboard {
-  constructor(params) {
+  // tslint:disable-next-line
```
Comment (ffflorian) [inline src/script/components/copyToClipboard.ts]: Why disable TSLint here?

Comment (lipis) [reply src/script/components/copyToClipboard.ts]: `'text' is declared but its value is never read.`

It's used in the template below, but TypeScript can't see that.. if you have anything better, we are open :)

Comment (lipis) [reply src/script/components/copyToClipboard.ts]: Travis is still complaining about it..

Comment (ffflorian) [reply src/script/components/copyToClipboard.ts]: Make it public :)

Comment (lipis) [reply src/script/components/copyToClipboard.ts]: It was that easy! Thanks

**Pull request merged**